applications/serial_lte_modem/src/main.c

```diff
@@ -269,24 +269,30 @@ static void handle_nrf_modem_lib_init_ret(void)
 	switch (ret) {
 	case 0:
 		return; /* Initialization successful, no action required. */
-	case MODEM_DFU_RESULT_OK:
+	case NRF_MODEM_DFU_RESULT_OK:
 		LOG_INF("MODEM UPDATE OK. Will run new firmware");
 		fota_stage = FOTA_STAGE_COMPLETE;
 		fota_status = FOTA_STATUS_OK;
 		fota_info = 0;
 		break;
-	case MODEM_DFU_RESULT_UUID_ERROR:
-	case MODEM_DFU_RESULT_AUTH_ERROR:
+	case NRF_MODEM_DFU_RESULT_UUID_ERROR:
+	case NRF_MODEM_DFU_RESULT_AUTH_ERROR:
 		LOG_ERR("MODEM UPDATE ERROR %d. Will run old firmware", ret);
 		fota_status = FOTA_STATUS_ERROR;
 		fota_info = ret;
 		break;
-	case MODEM_DFU_RESULT_HARDWARE_ERROR:
-	case MODEM_DFU_RESULT_INTERNAL_ERROR:
+	case NRF_MODEM_DFU_RESULT_HARDWARE_ERROR:
+	case NRF_MODEM_DFU_RESULT_INTERNAL_ERROR:
 		LOG_ERR("MODEM UPDATE FATAL ERROR %d. Modem failure", ret);
 		fota_status = FOTA_STATUS_ERROR;
 		fota_info = ret;
 		break;
+	case NRF_MODEM_DFU_RESULT_VOLTAGE_LOW:
```

Comment:
There will be doc change for this new cause, but we can do that after merging v2.3.0